docs/cloud/provision/aws/README.md

```diff
@@ -139,17 +129,17 @@ Now, although both networks are now connected, AWS doesn't create proper routes
 Click on `Edit routes` and then `Add route`. In the `Destination`, enter the CIDR of the EventStoreDB Cloud network. For the `Target`, choose the `Peering Connection` option.
 
 ::: card
-
 ![AWS route](./images/aws-network-route.png)
-
 :::
 
 The list of available peering connections will pop up. Select the recently created peering from the list and click on `Save routes`. The route table would then look like shown on the screenshot below.
 
 ::: card
-
 ![AWS route complete](./images/aws-network-route-done.png)
+:::
 
+::: tip Peering issues
+You might see the peering request getting stuck. There are several reasons for this to happen, like your cloud account quota or overlapping CIDR blocks. You can find all the necessary diagnostics in the `Event Console` in Event Store Cloud.
```

Comment:
**SUGGESTION:** Might be worth adding a link to the notification section added in the getting started
```suggestion
You might see the peering request getting stuck. There are several reasons for this to happen, like your cloud account quota or overlapping CIDR blocks. You can find all the necessary diagnostics in the [Event Console](docs/cloud/intro/quick-start.md#events-and-notifications) in Event Store Cloud.
```